composer.json

```diff
@@ -24,6 +24,7 @@
         "symfony/monolog-bundle": "^3.3",
         "symfony/swiftmailer-bundle": "^2.4",
         "symfony/symfony": "3.4.*",
+        "symfony/webpack-encore-bundle": "^1.6",
```

Comment:
This bundle comes with some configuration files that seem to be missing in this PR.
How about adding those, so we can use the power that this bundle provides?

Reply:
Can you be more specific. I know there are more encore settings to be configured in the `webpack.config.js` file. But the current setup seems to be pretty good for this project.

Reply:
The bundle has its own configuration: https://github.com/symfony/webpack-encore-bundle#configuration

This is required for the helper functions that come with the bundle to work